tests/.eslintrc

```diff
@@ -11,6 +11,7 @@
     "sinon": true,
   },
   "plugins": [
+    "jest",
```

Comment:
:+1:

Reply:
Oops. I messed up when rebasing. I guess that https://github.com/mozilla/addons-linter/pull/2639 can be closed now.

Reply:
I figured it'd be okay to just merge with this PR too.